code/datums/atmosphere/_atmosphere.dm

```diff
@@ -39,13 +42,13 @@
 		else
 			gastype = pick(restricted_gases)
 			amount = restricted_gases[gastype]
-			if(gaslist[gastype])
+			if(!HAS_TRAIT(SSstation, STATION_TRAIT_UNNATURAL_ATMOSPHERE) && gaslist[gastype])
```

Comment:
To be honest with you this is the part that worries me.

Reply:
removed the added pick chance

Reply:
Did you? After you showed me the output I'm not really fussed about this anymore, but isn't the chance still being upped to 80?

Reply:
I @'d you in discord with the results with the restriction chance up and you said that was okay so I put it back in, I made this comment before you asked me to test it 😳